impl/client/src/main/javascript/web/pentaho/i18n.js

```diff
@@ -148,4 +146,30 @@ define([
 
     throw new Error("[pentaho/messages!] Bundle path argument is invalid: '" + bundlePath + "'.");
   }
+
+  function getReminderUrlRegx() {
+    var defaultSchemePorts = {
+      "http:": 80,
+      "https:": 443
+    };
+
+    var url = env.server.root;
+
+    var origin = url.origin;
+    var port = url.port;
+    var pathname = url.pathname;
+
+    var isDefaultPort = port == null || port.length === 0;
+    if ( isDefaultPort ) {
+      var scheme = url.protocol;
+      var host = url.host;
+
+      port = defaultSchemePorts[scheme];
+
+      origin = scheme + "//" + host + ":" + port;
+      url = origin + pathname;
+    }
+
+    return new RegExp("^(?:" + url + "|" + origin + "|" + pathname + ")(.*)");
```

Comment:
It might be a good idea to escape `url`, `origin` and `pathname` before concatenation.
See https://github.com/benjamingr/RegExp.escape/blob/master/polyfill.js.